src/core/api/link_metrics_api.cpp

```diff
@@ -74,6 +74,7 @@ otError otLinkMetricsConfigForwardTrackingSeries(otInstance *
         static_cast<const Ip6::Address &>(*aDestination), aSeriesId, aSeriesFlags, aLinkMetricsFlags);
 }
 
+#if OPENTHREAD_CONFIG_MLE_LINK_METRICS_INITIATOR_ENABLE 
 otError otLinkMetricsConfigEnhAckProbing(otInstance *                               aInstance,
```

Comment:
In `link_metrics.hpp` can we add a comment to indicate that this API requires `OPENTHREAD_CONFIG_MLE_LINK_METRICS_INITIATOR_ENABLE`?

Reply:
This should do the trick:
https://github.com/openthread/openthread/blob/b726ff555634e8770547b6c6826f6c8e19f0f4ac/include/openthread/link_metrics.h#L202